ILTransform/ILRewriter.cs

```diff
@@ -556,7 +556,7 @@ private void RewriteFile(string source)
                                 }
                             }
                             int identEnd = charIndex;
-                            string sourceName = Path.GetFileNameWithoutExtension(source);
+                            TestProject.GetKeyNameRootNameAndSuffix(_testProject.TestProjectAlias, out _, out string sourceName, out _);
```

Comment:
I think this might need to be `RelativePath` and not `TestProjectAlias`.  The latter has already had `GetFileNameWithoutExtension` run on it, so with something like a.b.csproj, this would pull out a "second extension".

Reply:
Good catch